01-node-tutorial/answers/prompter.js

```diff
@@ -20,18 +20,24 @@ const getBody = (req, callback) => {
   });
 };
```

Comment:
This doesn't seem to compile for me.

Reply:
I just tried it and it works just fine, no idea how to fix it, since it works on my end. Any ideas?

Reply:
It is now working for me, sorry that may have been due to the way I was trying to run it. All good!